### PR TITLE
#131 client ip adding dialogue

### PR DESCRIPTION
**Pull request by arisb1470**
Navigate to the dashboard. Test the "Internes Projekt hinzufügen" button and the entire dialogue that subsequently opens.

### CODE REVIEW

File: client/src/ts/members/Dashboard.tsx [hunk 1]
```diff
@@ -1,9 +1,266 @@
...
+    return (
+      <div>
+        <Button variant="contained" onClick={handleClickOpen}>
+          Internes Projekt hinzufügen
+        </Button>
+        <Dialog open={open} onClose={handleClose}>
+          <DialogTitle>Informationen zum internen Projekt</DialogTitle>
+          <DialogContent>
+            <Box sx={{ flexGrow: 1 }}>
+              <Grid container spacing={1}>
```
Comment (JanMueller31) [inline client/src/ts/members/Dashboard.tsx:78]: Align labels with inputfields

File: client/src/ts/members/Dashboard.tsx [hunk 2]
```diff
@@ -1,9 +1,266 @@
...
+                </Grid>
+              </Grid>
+            </Box>
+            <Grid item xs={12} sm={12} md={12} lg={12}>
+              <Grid container spacing={1}>
+                <Grid item xs={12} sm={12} md={12} lg={12}>
+                  <Typography>Projektmitglieder:</Typography>
+                </Grid>
+
+                <Grid container spacing={1}>
```
Comment (JanMueller31) [inline client/src/ts/members/Dashboard.tsx:113]: Add margin between Projektmitglieder label and user inputfields

**Review by JanMueller31 — CHANGES_REQUESTED** [review]
Align labels and input fields and add margin under "Projektmitglier"

**Comment by arisb1470** [comment]
Added to #127

**Pull request closed without merging**